util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -367,6 +367,9 @@ macro (seqan_setup_library)
                  DESTINATION share/doc/seqan)
         # Install pkg-config file, except on Windows.
         if (NOT CMAKE_SYSTEM_NAME MATCHES Windows)
+            # Find packages and set flags appropriately
+            set (SEQAN_PKGCFG_DEPS "${SEQAN_PKGCFG_DEPS} zlib")
+            set (SEQAN_PKGCFG_LIBS "${SEQAN_PKGCFG_LIBS} -lbz2")
```

Comment:
Sure. These are hard-coded in essense, but I can just move their definition to the pkg-config file.